### PR TITLE
Create Backend Boilerplate

### PR DESCRIPTION
**Pull request by alkhatibmahmoud**
# Description

Created a boiler plate for the backend 

## Type of change
Initial commit for backend 

### Please add ticket link: https://trello.com/c/rxRDG4vt

### CODE REVIEW

File: backend/.env
Comment (serajsas) [inline backend/.env:1]: Out of curiosity, can we connet to local mongodb instance for example? Maybe through docker

File: backend/controllers/userController.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// userController.js
+const UserModel = require('../models/userModel');
+
+// Example controller function
+const getUsers = async (req, res) => {
+    try {
+        const users = await UserModel.find();
```
Comment (serajsas) [inline backend/controllers/userController.js:7]: How is the UserModel going to know which user to fetch  without passing any params to it

File: backend/.env
Comment (alkhatibmahmoud) [reply backend/.env:1]: We can create an instance of a cluster on Atlas and connect it.

File: backend/controllers/userController.js (hunk 1 above)
Comment (alkhatibmahmoud) [reply backend/controllers/userController.js:7]: This is just a boiler plate to set general structure of the code directories. Its a stub for all what matters.

**Review by serajsas — APPROVED** [review]

**Pull request merged**